db/migrate/20160526123633_add_userrole_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserroleToUser < ActiveRecord::Migration
+  def change
+     add_column :users, :user_role, :integer
```

Comment:
rename column just to `role`. `name` column could be splitted between 2 columns `first_name` and `last_name`